backend/framework/qlf/dashboard/bokeh/qainteg/main.py

```diff
@@ -40,15 +40,30 @@ def __init__(self, process_id, arm, spectrograph):
 
     def load_qa(self):
         cam = self.selected_arm+str(self.selected_spectrograph)
+        # try:
```

Comment:
If you are not going to use it just delete it. We can recover code from git history